pkg/controller/statefulset/stateful_set_utils.go

```diff
@@ -149,7 +149,6 @@ func getPersistentVolumeClaims(set *apps.StatefulSet, pod *v1.Pod) map[string]v1
 		claim := templates[i]
 		claim.Name = getPersistentVolumeClaimName(set, &claim, ordinal)
 		claim.Namespace = set.Namespace
-		claim.Labels = set.Spec.Selector.MatchLabels
```

Comment:
This line has come since https://github.com/kubernetes/kubernetes/pull/40838/files#diff-2ad3f4a4088a7249756d1b81af193493R147 which was a huge change of refactoring.
On review history, there was not comment related to this code. And the above explanation of this method(line 141-143) just mentions the name of claim, not the label.
I'm still not sure this code is intentional or not.. I hope the specialists can help this.

Reply:
@oomichi Thank you for your review. Currently, I have not found the labels of pvc used in anywhere of statefuleSets. To ensure the result intentional, I would also invite the author of these code to review it @kow3ns

Reply:
This code is intentional to allow the statefulset to find the PVCs it creates.